database/migrations/2016_08_15_182521_rename_kudos_table_to_reactions_table.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+
+class RenameKudosTableToReactionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::rename('kudos', 'reactions');
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        //
```

Comment:
Should probably have a drop function here that renames the `reactions` table to the `kudos` table. This will allow things to stay consistent with rolling back migrations.